panaroo/1.2.10/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM mambaorg/micromamba:0.22.0 as app
+
+# Version arguments
+# ARG variables only persist during build time
+ARG PANAROO_SOFTWARE_VERSION="1.2.10"
+
+# build and run as root users since micromamba image has 'mambauser' set as the $USER
+USER root
+# set workdir to default for building; set to /data at the end
+WORKDIR /
+
+LABEL base.image="mambaorg/micromamba:0.22.0"
+LABEL dockerfile.version="1"
+LABEL software="Panaroo"
+LABEL software.version=$PANAROO_SOFTWARE_VERSION}
+LABEL description="An updated pipeline for pangenome investigation"
+LABEL website="https://github.com/gtonkinhill/panaroo"
+LABEL license="https://github.com/gtonkinhill/panaroo/blob/master/LICENSE"
+LABEL maintainer1="Kevin Libuit"
+LABEL maintainer1.email="kevin.libuit@theiagen.com"
+
+# install dependencies; cleanup apt garbage
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ wget \
+ ca-certificates \
+ procps && \
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# Create Panaroo conda environment called panaroo-env from bioconda recipe
+# clean up conda garbage
+RUN micromamba create -n panaroo-env -c conda-forge -c bioconda -c defaults panaroo=${PANAROO_SOFTWARE_VERSION} && \
```

Comment:
Can you install in the base environment or does it need to be involved in a separate conda environment?

I'm curious if `ENV ENV_NAME="panaroo-env"` can be dropped.

Reply:
micromamba folks even recommend using the base env: https://github.com/mamba-org/micromamba-docker#quick-start

Reply:
change implemented